optuna/samplers/tpe/sampler.py

```diff
@@ -135,9 +135,16 @@ def sample_independent(self, study, trial, param_name, param_distribution):
             return self._random_sampler.sample_independent(
                 study, trial, param_name, param_distribution
             )
-
         below_param_values, above_param_values = self._split_observation_pairs(values, scores)
 
+        # Parzen estimator construction requires at least one observation or a priror.
+        n_below = len(below_param_values)
+        n_above = len(above_param_values)
+        if not self._parzen_estimator_parameters.consider_prior and (n_below == 0 or n_above == 0):
+            return self._random_sampler.sample_independent(
+                study, trial, param_name, param_distribution
+            )
+
```

Comment:
Is this fall-back consistent with the hyperopt implementation? cc: @HideakiImamura

Reply:
I have looked through the HyperOpt code.
It seems that HyperOpt does not have an option to disable to use a prior.
Therefore, HyperOpt has no fall-back for a situation like this.

The current fall-back is the same for the case: n < self._n_startup_trials.
https://github.com/optuna/optuna/blob/master/optuna/samplers/tpe/sampler.py#L139

Another idea is simply ignoring self._parzen_estimator_parameters.consider_prior flag and enabling a prior in this situation.

Reply:
I agree with @y0z. The hyperopt does not have any option to disable a prior. 
I think ignoring `consider_prior` flag is good idea in this situation, since it is consistent with normal situation.